File: pinyougou_parent/pinyougou_pay_interface/src/main/java/com/pinyougou/pay/service/PayService.java
package com.pinyougou.pay.service;

import java.util.Map;

public interface PayService {
    public Map<String,Object> createNative(String out_trade_no,String total_fee) throws Exception;
    public Map<String,String> queryPayStatus(String out_trade_no) throws Exception;
}

File: pinyougou_parent/pinyougou_pay_web/src/main/webapp/js/service/payService.js
//服务层
app.service('payService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.createNative=function(){
		return $http.get('pay/createNative.do');
	}

	this.queryPayStatus=function (out_trade_no) {
		return $http.get('pay/queryPayStatus.do?out_trade_no='+out_trade_no);
    }
});
